ftw/simplelayout/browser/provider.py

```diff
@@ -26,13 +26,21 @@ class BaseSimplelayoutExpression(object):
     fallbackview = ViewPageTemplateFile('templates/render_block_error.pt')
     structure = ViewPageTemplateFile('templates/structure.pt')
 
+    @property
+    def one_layout_one_column(self):
+        return [
+            {"cols": [{"blocks": []}
+                      ]
```

Comment:
I'd either add a trailing comma or move the closing bracket to the line above. Sorry, a little bit picky but it just popped into my eyes :wink: